backend/app/helpers/spree/admin/stores_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module Admin
+    module StoresHelper
+      def selected_checkout_zone(store)
+        return store.checkout_zone_id unless store.checkout_zone_id.nil?
+        return checkout_zone('No Limits').id if Spree::Config[:checkout_zone].nil?
+
+        return checkout_zone(Spree::Config[:checkout_zone]).id unless Spree::Config[:checkout_zone].nil?
```

Comment:
Won't an error be raised if  no `Spree::Zone` is found with requested name?

Reply:
could be, if someone intentionally will remove `No Limits` zone, or didn't setup preference in Spree config file with name `checkout_zone`. 

i'm updating field `checkout_zone_id` in existing stores (in case of upgrading Spree) during migration, I assume most of existing stores have already set `checkout_zone` because they have to paying somewhere taxes or creating zone `No Limits` in new instance of Spree store. On frontend we displaying all countries if `checkout_zone` isn't set.

Reply:
@nnande atm I see this kind solution where if some result will be `nil` I will convert to `-`0`, on fronted will be displayed next existing Zone https://github.com/spree/spree/pull/10529/commits/71a1b167c3c16ff1e71331a9b47787ae07167f87

Reply:
or, if we add html attribute `include_blank: true` this field will stay empty when `selected_checkout_zone` return `0`